cockroachdb/values.yaml

```diff
@@ -265,6 +265,15 @@ statefulset:
   securityContext:
     enabled: true
 
+  serviceAccount:
+    # Specifies whether this ServiceAccount should be created.
+    create: true
+    # The name of this ServiceAccount to use.
+    # If not set and `create` is `true`, then a name is auto-generated.
```

Comment:
Is the auto generated name `default`? If so can we just say that instead of `auto-generated`?

Reply:
I have added one more line of comment to make it more readable.